streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -117,7 +118,7 @@
      * });
      * }</pre>
      * <p>
-     * The provided {@link KeyValueMapper} must return a {@link KeyValue} type and the return value must not be {@code null}.
+     * The provided {@link KeyValueMapper} must return a {@link KeyValue} type and must not return {@code null}.
```

Comment:
I would not change anything with regard to `null`: if users want to change "cardinality" they should use "flatMap". If we want to do some reasoning about performance, it's good to know that map does not change cardinality: one in, one out.